File: videojuegoDeChinpokomon/src/Ataque/PomadaWassington.java
package Ataque;

import Chimpokomon.Chimpokomon;

public class PomadaWassington extends Ataque {
    public PomadaWassington(Integer dañoValor, String nombre) {
        super(dañoValor, nombre);

    }

    @Override
    public void dañar(Chimpokomon chipoAtacante, Chimpokomon chipoEnemigo) {

        if (chipoAtacante.tieneVentajaSobre(chipoEnemigo)) {
            chipoAtacante.nivelDeVida += 6;
            // System.out.printf(chimpokomonAtacante.getNombre() + " es un robot y " +
            // chinpokomon.getNombre()
            // + " es un animal, por lo tanto " + chimpokomonAtacante.getNombre()
            // + " recupera 6 puntos de vida.\n");
        } else {
            chipoAtacante.nivelDeVida += 5;
            // System.out.println(chimpokomonAtacante.getNombre() + " se cura 5 puntos de
            // vida");
        }

    }
}

File: ChimpokomonPython/ModuloSRC/Ataque/PomadaWassington.py
from Ataque.Ataque import Ataque

class PomadaWassington(Ataque):
    def __init__(self):
            Ataque.__init__(self, 1, "PomadaWassington")

    def dañar(self, chimpokomonAtacante, chimpokomonEnemigo):
        if(chimpokomonAtacante.tieneVentajaSobre(chimpokomonEnemigo)):
            chimpokomonAtacante.nivelDeVida(chimpokomonAtacante.nivelDeVida + 6)
        else:
            chimpokomonAtacante.nivelDeVida(chimpokomonAtacante.nivelDeVida + 5)

